app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -95,15 +95,13 @@
 
   <% if prefix && suffix %>
     <%= tag.div class: "gem-c-input__wrapper" do %>
-      <%= tag.span prefix, class: "gem-c-input__prefix", aria: { hidden: true } %>
-      <%= input_tag %>
-      <%= tag.span suffix, class: "gem-c-input__suffix", aria: { hidden: true } %>
+      <%= tag.span prefix, class: "gem-c-input__prefix", aria: { hidden: true } %><%= input_tag %><%= tag.span suffix, class: "gem-c-input__suffix", aria: { hidden: true } %>
```

Comment:
Presumably you've moved these three lines together for a reason... is it possible to document this somehow (maybe with a comment?) if splitting them out again breaks something?

Reply:
Yep, will add a comment above it.